src/App.js

```diff
@@ -76,9 +76,25 @@ export default class App extends React.Component {
   }
 
   async componentDidMount() {
+    let urlParams = this.parseUrlParams()
+    if (urlParams['schemaUrl']) { 
+      console.log("SSSS", urlParams['schemaUrl'])
+      this._handleUrlChange(urlParams['schemaUrl'])
+    }
     setTimeout(this.refreshEditorData, 1000);
   }
 
+  parseUrlParams(){
+    let url = window.location.href
```

Comment:
Should use `window.location.search`
And use this to parse the url: https://www.npmjs.com/package/qs ... 20mil downloads per week.